pallets/asset/src/lib.rs

```diff
@@ -408,6 +409,7 @@ decl_storage! {
         config(classic_migration_tickers): Vec<ClassicTickerImport>;
         config(classic_migration_tconfig): TickerRegistrationConfig<T::Moment>;
         config(classic_migration_contract_did): IdentityId;
+        config(reserved_country_currency_codes): Vec<Vec<u8>>;
```

Comment:
I'd prefer the genesis config to be more strongly typed, i.e. `Vec<Ticker>`, and to move the conversions to the genesis configs.